doc/how_to/waveform_extractor_to_sorting_analyzer.rst

```diff
@@ -0,0 +1,566 @@
+            analyzer.get_saved_extension_names()
+
+You can now also check which extensions are currently loaded *in memory*. The WaveformExtractor
+checks the local folder *and* the memory:
+
+.. grid:: 2
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            wvf_extractor.get_available_extension_names()
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            analyzer.get_loaded_extension_names()
+
+If there is an extensions which is saved but not yet loaded you can load it:
+
+.. grid:: 2
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            wvf_extractor.load_extension(
+                extension_name="spike_amplitudes"
+            )
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            analyzer.load_extension(
+                extension_name="spike_amplitudes"
+            )
+
+You can also check if a certain extension is loaded
+
+.. grid:: 2
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            wvf_extractor.has_extension(
+                extension_name="spike_amplitudes"
+            )
+
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            analyzer.has_extension(
+                extension_name="spike_amplitudes"
+            )
+
+You can delete extensions. Note that if you delete a parent, all of its children
+will be deleted too. We'll now delete ``templates`` from the SortingAnalyzer and ``spike_amplitudes`` from our WaveformExtractor.
+
+.. grid:: 2
+
+    .. grid-item::
+
+          .. code-block:: python
+
+            wvf_extractor.delete_extension(
+                extension_name="spike_amplitudes"
+            )
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            # This also deletes any parents
+            # such as spike_amplitudes
+            analyzer.delete_extension(
+                extension_name="templates"
+            )
+
+Once you have computed an extension, you often want to look at the data associated with it.
+This has been standardized for the ``SortingAnalyzer`` object, through the ``get_data`` method.
+The retrieval methods for the ``WaveformExtractor`` object were less uniform, and depended
+on which extension you were interested in. We won't list them all here.
+
+.. grid:: 2
+
+    .. grid-item::
+
+          .. code-block:: python
+
+            wv_data = wvf_extractor.get_waveforms(
+                unit_id=0
+            )
+
+            ul_data = compute_unit_locations(
+                waveform_extractor=wvf_extractor
+            )
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            wv = analyzer.get_extension(
+                extension_name="waveforms"
+            )
+            wv_data = wv.get_data()
+            ul = analyzer.get_extension(
+                extension_name="unit_locations"
+            )
+            ul_data = nl.get_data()
+
+You can also access the parameters used in the extension calculation, which is very simply for the new SortingAnalyzer:
```

Comment:
```suggestion
You can also access the parameters used in the extension calculation, which is very simple for the new SortingAnalyzer:
```
This one too. Probably just forgot to pull from the github changes so these were overwritten when you pushed.